server/test/endpoint_admin_personas_roles.test.js

```diff
@@ -0,0 +1,76 @@
+const { describe, it, before, after } = require('mocha')
+const fetch = require('node-fetch')
+const assert = require('assert')
+
+const { getAdmin1IdPToken, getTestUser1IdPToken } = require('./lib/tokens')
+const { getServerUrl } = require('./lib/urls')
+const { usherDb } = require('../../database/layer/knex')
+
+
+describe('Admin Personas Roles', () => {
+  let requestHeaders
+  let testPersonaKey
+  const url = `${getServerUrl()}`
```

Comment:
@k1ch  just a style question, is there a reason to use a template string here as opposed to just the function `getServerUrl()` ?

Reply:
Thank you for pointing that out. I fixed it in the other files too.